dockstore-client/src/main/java/io/dockstore/client/cli/nested/WesLauncher.java

```diff
@@ -117,6 +134,54 @@ public static Workflow getWorkflowForEntry(WorkflowClient workflowClient, String
         return workflowClient.getWorkflowsApi().getPublishedWorkflowByPath(path, WorkflowSubClass.BIOWORKFLOW.toString(), null, version);
     }
 
+    /**
+     * Provisions a workflow's associated files in a temporary local directory
+     *
+     * @param workflowClient The WorkflowClient
+     * @param workflowEntry The workflow entry (i.e. github.com/myRepo/myWorkflow:version)
+     * @param descriptorType The type of descriptor for this workflow (WDL, CWL, etc...)
+     * @return A zipped File object
+     */
+    public static File provisionFilesLocally(WorkflowClient workflowClient, String workflowEntry, String descriptorType) {
+
+        // A temporary directory which will house all downloaded content
+        File tempDir;
+
+        try {
+            tempDir = Files.createTempDirectory(DOCKSTORE_ROOT_TEMP_DIR_PREFIX).toFile().getAbsoluteFile();
```

Comment:
deleteOnExit, see other comment, unless there's a reason to keep it around.

Reply:
Added: https://github.com/dockstore/dockstore-cli/pull/113/commits/73b5a0bbeb3d04bf035242170002d24af323df96